### PR TITLE
recognition api implementation

### PR DESCRIPTION
**Pull request by shailendrakanherkar18**

### CODE REVIEW

File: go-backend/db/recognition.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	CoreValuesID    int       `db:"core_values_id" json:"core_values_id"`
+	RecognitionText string    `db:"recognition_text" json:"recognition_text"`
+	RecognitionFor  int       `db:"recognition_for" json:"recognition_for"`
+	RecognitionBy   int       `db:"recognition_by" json:"recognition_by"`
+	RecognitionOn   time.Time `db:"recognition_on" json:"recognition_on"`
+}
+
+func (s *pgStore) CreateRecognition(ctx context.Context, recognition Recognition) (err error) {
+	createRecognitionQuery := `
+		INSERT INTO recognitions (
```
Comment (iamanujvrma) [inline go-backend/db/recognition.go:22]: You can decare all the queries as `const` on the top.

**Comment by anilmaurya** [comment]
This PR is dependent on #39 

Once #39 is merged , we will update this PR .

File: go-backend/db/recognition.go [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+			given_by,
+			given_at
+		)
+		VALUES ($1, $2, $3, $4, $5) returning id
+	`
+	showRecognitionQuery = "SELECT * FROM recognitions WHERE id=$1"
+	listRecognitionQuery = "SELECT * FROM recognitions ORDER BY given_at ASC"
+)
+
+func listRecognitionsFilterByOneFilterQuery(column_name string) string {
```
Comment (iamanujvrma) [inline go-backend/db/recognition.go:24]: this is not a good way to create dynamic queries, instead where parameters can be added dynamically on the basis of number of filters required through a loop

File: go-backend/db/recognition_test.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
+package db
```
Comment (iamanujvrma) [inline go-backend/db/recognition_test.go:1]: we have deprecated this strategy for writing test case, please refer https://github.com/joshsoftware/peerly/pull/110
for the new strategy

Comment (iamanujvrma) [reply go-backend/db/recognition_test.go:1]: Its already merged so the code is in master also

**Pull request merged**